docs/api/input.md

```diff
@@ -134,6 +134,14 @@ import Migration from '@site/static/usage/v7/input/migration/index.md';
 
 Ionic uses heuristics to detect if an app is using the modern input syntax. In some instances, it may be preferable to continue using the legacy syntax. Developers can set the `legacy` property on `ion-input` to `true` to force that instance of the input to use the legacy syntax.
 
+## Input Masking
```

Comment:
Can we add a section directing developers to Maskito's github for filing issues and to our forums for technical support? I'm thinking it could look something like the last two FAQs on https://ionic-docs-git-sp-maskito-ionic1.vercel.app/docs/angular/slides#faq